src/main/java/marquez/service/DatasourceService.java

```diff
@@ -76,7 +76,7 @@ public Datasource create(@NonNull ConnectionUrl connectionUrl, @NonNull Datasour
     }
   }
 
-  public List<Datasource> getAll(@NonNull Integer limit, @NonNull Integer offset) {
+  public List<Datasource> list(@NonNull Integer limit, @NonNull Integer offset) {
```

Comment:
Don't we want this to be `DatasourceService.getAll()`?

Reply:
Yes, sorry, misunderstood a comment on a different PR. Addressed that, and fixed this mis-naming.